packages/transport-redis/src/RedisTransport.js

```diff
@@ -9,6 +9,7 @@ import map from 'lodash/map';
 import size from 'lodash/size';
 import split from 'lodash/split';
 import Promise from 'bluebird';
+import uuidv1 from 'uuid/v1';
```

Comment:
is v1 is the fastest?

Reply:
v1 is 2.5 million rps compared to v4 which is 250k rps